node/bridge.ts

```diff
@@ -56,7 +56,7 @@ class DenoBridge {
     this.cacheDirectory = options.cacheDirectory ?? getPathInHome('deno-cli')
     this.debug = options.debug ?? false
     this.denoDir = options.denoDir
-    this.logger = options.logger ?? getLogger(undefined, options.debug)
+    this.logger = options.logger ?? getLogger(undefined, undefined, options.debug)
```

Comment:
nit: Might be nice to make `getLogger` receive an object so that we don't have to supply a bunch of `undefined`.

Totally not a blocker, though.